packages/session-replay-browser/src/session-replay.ts

```diff
@@ -74,6 +83,27 @@ export class SessionReplay implements AmplitudeSessionReplay {
     this.config.privacyConfig.unmaskSelector = dropInvalidSelectors(this.config.privacyConfig.unmaskSelector);
   }
 
+  private setupEventListeners = (teardown: boolean) => {
+    const globalScope = getGlobalScope();
```

Comment:
I'd rather we have a `teardownEventListeners` and a `setupEventListeners` function (which would call `teardownEventListeners`) than pass `teardown` as a param here, using `setupEventListeners` in the `shutdown` method looks confusing to me, wdyt?

Reply:
Yeah that SGTM, I had some troubles with naming here.